contracts/dummy/dummy.hpp

```diff
@@ -0,0 +1,113 @@
+/**
+ *  @file
+ *  @copyright defined in eos/LICENSE.txt
+ */
+
+#include <eoslib/eos.hpp>
+#include <eoslib/token.hpp>
+#include <eoslib/db.hpp>
+
+/**
+ *  @defgroup examplecontract Example Contract
+ *  @brief Example Contract
+ *  @ingroup contractapi
+ *
```

Comment:
Update doxygen with correct info. Also indicate what the purpose of this contract is for.